### ISSUE TITLE
Source range in error message too long in case of non ASCII symbols

### ISSUE THREAD

**Issue by fridis** [tokiwa-software/fuzion#3292]
Try this:
```
 > ./build/bin/fz -e 'x := "漢字" ≥≥≥≥≥≥ "ひらがな"' 

command line:1:13: error 1: Could not find called feature
x := "漢字" ≥≥≥≥≥≥ "ひらがな"
------------^^^^^^^^^^^^^^^^^^
Feature not found: 'infix ≥≥≥≥≥≥' (one argument)
Target feature: 'String'
In call: '≥≥≥≥≥≥'

one error.
```
The range indicator in the error message position should be just
```
------------^^^^^^
```

**Comment by simonvonhackewitz** [comment]
this seems to be resolved with #3342

**Comment by fridis** [comment]
> this seems to be resolved with #3342

Only if terminal escape codes are used, I guess. Try 

    FUZION_DISABLE_ANSI_ESCAPES=false  ./build/bin/fz -e 'x := "漢字" ≥≥≥≥≥≥ "ひらがな"'
